### ISSUE TITLE
Create pip package for genie_python

### ISSUE THREAD

**Issue by DominicOram** [ISISComputingGroup/IBEX#3571]
As a developer I would like to be able to get genie python into virtual environments more easily and so allow me to separate the requirements of different python projects within IBEX. The standard way to do this in python is to create a pip package that you can install.

# Acceptance Criteria
* I can pip install genie python

Note:
* This should not be put on PyPi and made public yet as the code is sensitive. Instead it should be on a internal share
* The package should endeavour to describe it's dependencies, where they are also easily installed via pip e.g. numpy but it's ok if some dependencies are not installed as part of the package e.g. epics base

**Comment by DominicOram** [comment]
See https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Python-dependencies for more info

**Comment by DominicOram** [comment]
Once this is done remove the `--system-site-packages` flag where virtual environments are set up in specific projects and replace with adding `genie_python` to the `requirements.txt`

**Comment by Tom-Willemsen** [comment]
https://github.com/ISISComputingGroup/IBEX/issues/8381 is effectively a duplicate of this ticket, but contains more detail, so closing this one.
